cmd/kubeadm/app/cmd/phases/kubelet.go

```diff
@@ -19,12 +19,13 @@ package phases
 import (
 	"fmt"
 
+	"github.com/golang/glog"
 	"github.com/spf13/cobra"
-
```

Comment:
leave this empty line here?

Reply:
gofmt automatically removes those lines...